checker/build.gradle

```diff
@@ -4,9 +4,16 @@ plugins {
 }
 sourceSets {
     main {
+        java {
+            // NO-AFU
+            exclude '**/org/checkerframework/checker/resourceleak/MustCallInference.java'
+        }
         resources {
             // Stub files, message.properties, etc.
             srcDirs += ['src/main/java']
+
+            // NO-AFU
+            exclude '**/org/checkerframework/checker/resourceleak/MustCallInference.java'
```

Comment:
I see we've mixed resources in with the Java code. Should this source set just exclude `**/*.java`? Or do we expect the source files to be included in the compiled jar?